ca_gazebo/launch/create_empty_world.launch

```diff
@@ -14,13 +14,23 @@
 		<arg name="verbose" value="true"/>
 		<arg name="world_name" value="$(find ca_gazebo)/worlds/$(arg env).world"/>
 	</include>
+	
+	<group if="$(eval str(arg('env'))=='empty')">
```

Comment:
Is this a fix that is not in the original repo? If so, consider creating a PR with the fix.